components/Products.tsx

```diff
@@ -0,0 +1,56 @@
+import { TEXT_MEDIUM_CLASS } from 'styles/globals';
+import { SiteLink } from './SiteLink';
+
+const products = [
+  {
+    title: 'Propel',
+    desc: 'The fastest way to get a decentralized autonomous agent to market. \n A subscription service blending human and software services, to suit any organization size.',
+  },
+  {
+    title: 'Olas Protocol',
+    desc: 'The permissionless blockchain protocol for registering, running and securing open-source, decentralized and ownable autonomous agents.',
+    href: 'https://olas.network/protocol',
+  },
+  {
+    title: 'Olas Stack',
+    desc: 'Open-source framework for developing decentralized and ownable autonomous agents.',
+    href: 'https://olas.network/stack',
+  },
+];
+
+export const Products = () => (
```

Comment:
can you please check this section on smaller sizes, it looks a bit different, like it has different background gradient for different sizes?
<img width="739" alt="image" src="https://github.com/user-attachments/assets/46c9d24c-8075-4fd0-a344-99cd0a2517af" />

Reply:
Can fix the background but what are ur thoughts about the layout? Currently it is also different and maintains the 3 col grid:
![image](https://github.com/user-attachments/assets/39bff640-b298-4814-aa8d-df6a2a3df554)

Reply:
columns are fine imo, thanks!